# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/rav1e/hotfix-older-rustc.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- rav1e-0.7.1/ivf/Cargo.toml.vanilla	2024-06-16 12:34:25.655000000 +0200
+++ rav1e-0.7.1/ivf/Cargo.toml	2024-06-16 12:33:01.523000000 +0200
@@ -8,4 +8,4 @@
 homepage = "https://github.com/xiph/rav1e"
 
 [dependencies]
-bitstream-io = "2"
+bitstream-io = "=2.3.0"
--- rav1e-0.7.1/Cargo.toml.vanilla	2024-06-16 12:29:22.383000000 +0200
+++ rav1e-0.7.1/Cargo.toml	2024-06-16 12:32:32.960000000 +0200
@@ -72,7 +72,7 @@
 
 [dependencies]
 arg_enum_proc_macro = "0.3.4"
-bitstream-io = "2"
+bitstream-io = "=2.3.0"
 cfg-if = "1.0"
 clap = { version = "4.4.11", optional = true, default-features = false, features = [
   "color",
